st2common/st2common/openapi.yaml

```diff
@@ -0,0 +1,3781 @@
+swagger: '2.0'
+
+info:
+  version: "1.0.0"
+  title: StackStorm API
+
+paths:
+  /api/v1/:
+    get:
+      operationId: st2api.controllers.root:root_controller.index
+      description: General API info.
+      responses:
+        '200':
+          description: General API info.
+          schema:
+            type: object
+            properties:
+              version:
+                type: string
+              docs_url:
+                type: string
+          examples:
+            application/json:
+              ref: 'core.local'
+              # and stuff
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+  /api/v1/actions:
+    get:
+      operationId: st2api.controllers.v1.actions:actions_controller.get_all
+      x-permissions: {{ PERMISSION_TYPE.ACTION_LIST }}
```

Comment:
Bit confused, I see `x-permissions` here but not for some other controllers (e.g. create action, etc.).

Is this intentional for some reason or it still needs to be finished?